packages/lib-core/src/runtime/PluginStore.ts

```diff
@@ -23,13 +18,6 @@ export type PluginStoreOptions = Partial<{
    * Default value: `true`.
    */
   autoEnableLoadedPlugins: boolean;
-
-  /**
-   * Post-process the plugin's extension objects.
-   *
-   * By default, no post-processing is performed on the extension objects.
-   */
-  postProcessExtensions: (extensions: LoadedExtension[]) => LoadedExtension[];
```

Comment:
Chatted with @vojtechszocs and this post process of extensions was not utilized and didn't really make sense to expose with postProcessManifest.